### DISCUSSION TITLE
How do I deploy Hyas/Doks as github.io page?

### DISCUSSION THREAD

**Discussion by ondvrsn** [gethyas/hyas#438] [Q&A] [answered]
Hi, 

all

I have looked up both documents of hyas and doks, I had no luck.
Is there any instruction about this that I couldn't find?
Or Is there any advice I could used to start with?
My local pages works just fine, but I am really frustrated that I can't deploy it.
Please help.

Thanks in advance

**Comment by h-enk** [comment, marked as answer]
Documentation will be updated shortly. For now, see: https://github.com/h-enk/doks-gh-pages

**Reply by ondvrsn** [reply]
Thank you for the quick help.
I will try this instruction!

**Reply by h-enk** [reply]
Just added this guide: [Deploy your Hyas Site to GitHub Pages](https://docs.gethyas.com/guides/deploy/github/)
